core/rawdb/schema.go

```diff
@@ -126,6 +126,10 @@ var (
 	trieNodeStoragePrefix = []byte("O") // trieNodeStoragePrefix + accountHash + hexPath -> trie node
 	stateIDPrefix         = []byte("L") // stateIDPrefix + state root -> state id
 
+	// Agg-Path-based storage scheme of merkle patricia trie.
+	aggTrieNodeAccountPrefix = []byte("m") // aggTrieNodeAccountPrefix + hexPath -> agg trie node
```

Comment:
following the trieNodeAccountPrefix(PathDB) specification, uppercase(M and G) letters should be more appropriate

Reply:
Fixed